### PR TITLE
 feat: add aws codecommit resource

### PR DESCRIPTION
**Pull request by AmalRichwin**
## Problem

Need to support of AWS CodeCommit https://github.com/tailwarden/komiser/issues/1175

## Solution

Added AWS CodeCommit support

## Changes Made

- Added AWS CodeCommit Support

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [x] Changes have been thoroughly tested
- [x] Any dependencies have been added to the project, if necessary

## Reviewers

@Azanul

### CODE REVIEW

**Comment by Azanul** [comment]
@AmalRichwin Could you rebase and resolve the conflicts?

File: providers/aws/codecommit/repositories.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			resources = append(resources, models.Resource{
+				Provider:   "AWS",
+				Account:    client.Name,
+				Service:    "CodeCommit",
+				ResourceId: *repository.RepositoryId,
+				Region:     client.AWSClient.Region,
+				Name:       *repository.RepositoryName,
+				Tags:       tags,
+				FetchedAt:  time.Now(),
+				Link:       fmt.Sprintf("https://%s.console.aws.amazon.com/codesuite/codecommit/repositories/%s/browse?region=%s", client.AWSClient.Region, *repository.RepositoryName, client.AWSClient.Region),
```
[the remark this answers is not in the document]
Comment (AmalRichwin) [reply providers/aws/codecommit/repositories.go:52]: yes url is correctly landing into aws code commit selected respository page.

File: providers/aws/codecommit/repositories.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import (
+	"context"
+	"fmt"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/codecommit"
+	// "github.com/aws/aws-sdk-go-v2/service/sts"
```
Comment (AvineshTripathi) [inline providers/aws/codecommit/repositories.go:12]:
```suggestion
```

Comment (AvineshTripathi) [reply providers/aws/codecommit/repositories.go:12]: after this change we can merge the PR thanks

Comment (AmalRichwin) [reply providers/aws/codecommit/repositories.go]: Removed the commented code. Ready for merge. I apologize for the oversight, Thanks!

**Review by AvineshTripathi — APPROVED** [review]

**Comment by AvineshTripathi** [comment]
@AmalRichwin can you do a quick rebase so that we can merge this

**Comment by AmalRichwin** [comment]
rebased the branch

**Pull request merged**